plugins/eddn.py

```diff
@@ -596,6 +602,14 @@ def plugin_stop() -> None:
 def journal_entry(  # noqa: C901
     cmdr: str, is_beta: bool, system: str, station: str, entry: MutableMapping[str, Any], state: Mapping[str, Any]
 ) -> Optional[str]:
+    if (ks := killswitch.get_disabled("plugins.eddn.journal")).disabled:
+        logger.warning(f"EDDN journal handler has been disabled via killswitch: {ks.reason}")
+        plug.show_error("EDDN journal handler disabled. See Log.")
+        return None
+    elif (ks := killswitch.get_disabled(f'plugins.eddn.journal.event{entry["event"]}')).disabled:
```

Comment:
Isn't this missing a `.`, i.e. it should be

`killswitch.get_disabled(f'plugins.eddn.journal.event.{entry["event"]}')).disabled`

?  I'm going to guess the other plugins will have the same issue.

Reply:
Whoops. Yes

Reply:
Fixed